src/components/Error/ErrorBoundary.tsx

```diff
@@ -1,4 +1,10 @@
-import { DEFAULT_ERROR_ACTION, DEFAULT_ERROR_HEADER, WidgetError } from 'errors'
+import {
+  DEFAULT_ERROR_ACTION,
+  DEFAULT_ERROR_HEADER,
+  DISMISS_ERROR_ACTION,
```

Comment:
I think this should be named DEFAULT_DISMISSABLE_ERROR_HEADER, etc. It is still used as a default (assuming it is only used when no other header is passed.).